alot/commands/thread.py

```diff
@@ -465,9 +465,15 @@ async def apply(self, ui):
         tags.difference_update({'inbox', 'sent', 'draft', 'killed', 'replied',
                                 'signed', 'encrypted', 'unread', 'attachment'})
         tags = list(tags)
+
+        draft = None
+        if 'draft' in self.message.get_tags():
+            draft = self.message
+
         # set body text
         mailcontent = self.message.get_body_text()
-        envelope = Envelope(bodytext=mailcontent, tags=tags)
+        envelope = Envelope(bodytext=mailcontent, tags=tags,
+                            previous_draft=draft)
```

Comment:
I don't quite see how this is picket up by my editor then in envelope mode?
Am I missing something or is then envelope -> edit command simply ignoring this?

Reply:
Your `$EDITOR`? Is that necessary? `SendCommand` and `SaveCommand` simply pick it up from this field in the envelope.

Reply:
Sorry, I was confused and only now understood the logic behind this patch.
Question: What if I answer "no" to the prompt to remove the draft, then create another draft and end up with two drafts, then send the last and say "yes" to the prompt. 
As far as I can see, the current logic would only remove the very last draft, not the first one?
If so, is that intentional?

I could not test this myself at the minute because I have problems installing the dev version with poetry.. :D